### DISCUSSION TITLE
Contour_plot within wandb

### DISCUSSION THREAD

**Discussion by andife** [optuna/optuna#4993] [Q&A] [answered]
I am currently evaluating optuna with wandb and was wondering if it is possible to use the interactive Contour_plots directly in the wandb interface? Or display them statically? I find these particularly helpful when there are various hyperparameters to optimize.

**Comment by nzw0301** [comment, marked as answer]
According to https://docs.wandb.ai/guides/track/log/plots#matplotlib-and-plotly-plots, you can do both ways.
